docs/troubleshooting/cpuusage.rst

```diff
@@ -0,0 +1,112 @@
+.. _cpuusage:
+
+**************
+CPU Load/Usage
+**************
+
+Are there any command or tools in vpp to show real cpu and memory usage at runtime?
```

Comment:
suggestion:
FD.io VPP has various commands and tools that can help users see real CPU and memory usage at runtime.

Reply:
Looks great !